media/js/firefox/android.js

```diff
@@ -312,30 +314,14 @@
             'screencast': 'Take it to the big screen'
         }[$(this).attr('id')];
 
-        $(this).on('expand', '[role="tabpanel"]', function() {
```

Comment:
Since there's now nothing going on in this `each` loop, best to remove it entirely.